src/test/unit_tests/jest_tests/test_nc_account_cli.test.js

```diff
@@ -23,6 +23,13 @@ const timeout = 50000;
 const config = require('../../../../config');
 const config_fs_account_options = { show_secrets: true, decrypt_secret_key: true };
 
+const quoted_type = Object.freeze({
+    SPACE_ONLY: "s", // space only
+    QUOTE_SPACE: "qs", // quote then space
+    SPACE_QUOTE: "sq", // space then quote
+    NONE: "none" // none
```

Comment:
Do you need the "NONE"? I think it goes to the defaults.

Reply:
Yes we can remove it. Thanks